.github/workflows/pr-e2e-tests.yaml

```diff
@@ -13,6 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
```

Comment:
We do not want to do the same for unit tests?

Reply:
Added in https://github.com/neo4j/neo4j-genai-python/pull/67/commits/43e1cf56cf7fd1920693fda75f5c25f798f38a03